README.md

```diff
@@ -177,6 +189,9 @@ It is possible to specify a `releaseVersion` and `developVersion` on command lin
 grunt sg_release --releaseVersion 1.0.0 --developVersion 1.0.1-rc
 ```
 
+## Creating a hotfix
+A hotfix follows the same flow as a release but it usually branches off the master branch. All you have to do is to first checkout the master branch, then call the sg_release task. 
+
```

Comment:
well... from your previous comment in #6 :

> Meaning, instead of starting with branching of develop, rather branch of master, pause in the middle, do my hotfix changes on this hotfix branch and then merge to back to master + merge to develop?

I was under the impression that anyone that would like to create a **hotfix** would follow the same workflow but if that is not the case then don't worry.

The **docs** update is looking really good :+1: